### PR TITLE
update dbms.cluster.discovery.resolver_type

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-877.surge.sh

Updated pages:
- [SHOW SETTINGS](https://neo4j-docs-cypher-877.surge.sh/cypher-manual/5/clauses/listing-settings)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-877.surge.sh

Updated pages:
- [SHOW SETTINGS](https://neo4j-docs-cypher-877.surge.sh/cypher-manual/5/clauses/listing-settings)

File: modules/ROOT/pages/clauses/listing-settings.adoc [hunk 1]
```diff
@@ -213,7 +213,14 @@ LIMIT 10
 
 | "dbms.cluster.discovery.resolver_type"
 | "LIST"
-| "Configure the discovery type used for cluster name resolution"
+| "Configure the resolver type that the discovery service uses for determining who should be part of the cluster.
+Valid values are `LIST`, `SRV`, `DNS`, and `K8S`:
```
Comment (renetapopova) [inline modules/ROOT/pages/clauses/listing-settings.adoc]: I think you need a new line here.
```suggestion
Valid values are `LIST`, `SRV`, `DNS`, and `K8S`:
+
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-877.surge.sh

Updated pages:
- [SHOW SETTINGS](https://neo4j-docs-cypher-877.surge.sh/cypher-manual/5/clauses/listing-settings)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-877.surge.sh

Updated pages:
- [SHOW SETTINGS](https://neo4j-docs-cypher-877.surge.sh/cypher-manual/5/clauses/listing-settings)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**